JavaScript/packages/recognizers-text/src/culture.ts

```diff
@@ -37,10 +39,10 @@ export class Culture {
   static mapToNearestLanguage(cultureCode: string): string {
     if (cultureCode !== undefined) {    
       cultureCode = cultureCode.toLowerCase();
-      var supportedCultureCodes = Culture.getSupportedCultureCodes();
+      let supportedCultureCodes = Culture.getSupportedCultureCodes();
       
       if (supportedCultureCodes.indexOf(cultureCode) < 0) {
-        var culturePrefix = cultureCode.split('-')[0].trim();
+        let culturePrefix = cultureCode.split('-')[0].trim();
 
         supportedCultureCodes.forEach(function(supportedCultureCode) {
           if (supportedCultureCode.startsWith(culturePrefix)) {
```

Comment:
The behaviour here is not correct for en-*, right?
Also, there are no enabled tests for EnglishOthers in JS in the PR. Can you enable them?